example/discord_bot/basic_prefix.py

```diff
@@ -0,0 +1,34 @@
+import discord
+
+import vacefron
+
+from discord.ext import commands
+
+intents= discord.Intents.default()
```

Comment:
Please use the required intents for this example.
```suggestion
intent = discord.Intents(guilds=True, messages=True, message_content=True)
```